core/src/main/scala/codesearch/core/index/indexer/HaskellIndexer.scala

```diff
@@ -0,0 +1,11 @@
+package codesearch.core.index.indexer
+
+import cats.effect.{ContextShift, Sync}
+import codesearch.core.index.directory.СindexDirectory.HaskellCindex
```

Comment:
Please fix the Cyrillic C in "CindexDirectory" while you're at it.

Reply:
Thanks for finding mistake.